pkg/rtc/room.go

```diff
@@ -47,9 +48,10 @@ type broadcastOptions struct {
 type Room struct {
 	lock sync.RWMutex
 
-	protoRoom *livekit.Room
-	internal  *livekit.RoomInternal
-	Logger    logger.Logger
+	protoRoom  *livekit.Room
```

Comment:
Is this still needed? Can `protoProxy` be the one true value?

Reply:
yes they are. protoProxy is only holding a cached copy. it's not designed to be modified directly.